ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    mu_threshHigh = 1.0
</pre><p>per Python naming conversions, snake_case is used for variable names. So <code>mu_threshHigh</code> -&gt; <code>mu_threshold_high</code>, etc.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>